CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to
 
 ### Changed
 
+- In cli, change `push` to `write` and `fetch` to `read`
```

Comment:
1. It's often a good idea to flag breaking changes.
2. Also, maybe we need to prefix change's category, _i.e._ _Helm_ or _CLI_ etc. WDYT?

```suggestion
- CLI: change `push` to `write` and `fetch` to `read` [BC]
```

Reply:
I agree